src/Serializer/AbstractItemNormalizer.php

```diff
@@ -481,4 +491,44 @@ protected function normalizeRelation(PropertyMetadata $propertyMetadata, $relate
 
         return $iri;
     }
+
+    /**
+     * Populates the object if allowed to and not set already. If you pass identifiers, it will search for them, otherwise
+     * fall back to the identifiersextractor.
+     *
+     * @param $data
+     * @param $class
+     * @param array $context
+     * @param array $identifiers
+     */
+    protected function populateObjectIfAllowed($data, $class, array $context = [])
+    {
+        if (isset($context[self::OBJECT_TO_POPULATE])) {
+            return;
+        }
+
+        $identifiers = $this->getIdentifiersForDenormalization($class);
+
+        if (0 !== count(array_intersect(array_keys($data), $identifiers))) {
```

Comment:
I'm really not sure about this but maybe that performances would be better with a simple foreach test:

```
foreach ($identifiers as $identifier) { if (!isset($data[$identifier])) { return false; } }
return true;
```

Reply:
Also it should be `\count`

Reply:
Can be simplified to `if (\count(array_intersect(array_keys($data), $identifiers))`